api/src/main/java/net/md_5/bungee/api/boss/BossBar.java

```diff
@@ -135,6 +133,14 @@
      */
     Collection<BossBarFlag> getFlags();
 
+    /**
+     * Adds a flag to the boss bar
+     *
+     * @param flag the flag you wish to add
+     * @return if the was was not present before
+     */
+    boolean addFlag(BossBarFlag flag);
```

Comment:
Not needed as addFlags requires a `BossBarFlag...` which if you don't want to add more flags, you don't